client/my-sites/plugins/sidebar/index.tsx

```diff
@@ -34,10 +37,23 @@ const PluginsSidebar = ( { path, isCollapsed }: Props ) => {
 					label={ translate( 'Marketplace' ) }
 					tooltip={ isCollapsed && translate( 'Marketplace' ) }
 					selected={
-						path.startsWith( '/plugins' ) && ! path.startsWith( '/plugins/scheduled-updates' )
+						path.startsWith( '/plugins' ) &&
```

Comment:
This works when we're at the main `/plugins/manage` screen. The only remaining piece is to ensure that when we navigate inside a plugin, we also get the `Manage Plugins` enabled instead of `Marketplace`. 

Maybe we can use the `Installed Plugins` link to logic decide that?

https://github.com/user-attachments/assets/d7751b66-9cb3-4969-a2a8-d49431114fe8

Reply:
@paulopmt1 I'm wondering if how it works now is the correct way. Technically when you select the plugin it really navigates to the `Marketplace` to show the plugins details, then the `Installed plugins` breadcrumb navigates back to `plugins/manage`.

The main issue I fix from my latest commit is that if you select a filter like `Inactive` it would highlight the `Marketplace` as selected.

Reply:
> Technically when you select the plugin it really navigates to the Marketplace to show the plugins details, then the Installed plugins breadcrumb navigates back to plugins/manage.

Good point. 
@nuriapenya, can we get a design review of this behavior? How would you see us implementing this navigation?

Reply:
Oh, I see. Although I understand technically that when we load the marketplace page, the users come from `Manage plugins`, so the expectation is that "going back" will lead there. The highlighted item in the navigation should still be `Manage Plugins`. Otherwise, it's very confusing.

I also observed one thing:

In the navigation, we say `Manage Plugins`, but on the page header is `Installed Plugins`.

I'd keep consistency for both and say, `Installed Plugins` in both.

> Icon needs to be changed

@vykes-mac The one you have works quite well!